tests/src/test/java/alluxio/shell/command/ChownCommandTest.java

```diff
@@ -55,4 +115,50 @@ public void chownRecursive() throws IOException, AlluxioException {
     owner = mFileSystem.getStatus(new AlluxioURI("/testDir/testFile")).getOwner();
     Assert.assertEquals("user2", owner);
   }
+
+  /**
+   * Run chown command to change owner and group of a path.
+   *
+   * @param command the command to run
+   * @param expectedReturnValue return value expected from running the command
+   * @param expectedCommandOutput command output expected from running the command
+   * @param expectedOwner expected owner of the path
+   * @param expectedGroup expected group of the path
+   * @throws Exception
+   */
+  private void runChownOwnerAndGroup(String[] command, int expectedReturnValue,
```

Comment:
This does two things: (1) check that the command produces the expected return value and string and (2) check that the owner and group are set as expected. It would make sense to add the first functionality to `AbstractAlluxioShellTest` since many shell tests will want to run a command and check the return value and output. Once we have that method, we won't need this one any more and callers can instead run something like
```java
testCommand(-1, expectedCommandOutput, "chown", newOwner + ":" + nonexistGroup, "/testFile")
checkPathOwnerAndGroup("/testFile", originalOwner, originalGroup);
```